src/engine/experimental/eip7547.md

```diff
@@ -0,0 +1,101 @@
+# Engine API -- EIP7547
+
+Engine API changes introduced in EIP7547.
+
+This specification is based on and extends [Engine API - Cancun](./cancun.md) specification.
+
+## Table of contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Constants](#constants)
+- [Structures](#structures)
+  - [InclusionListSummaryEntryV1](#inclusionlistsummaryentryv1)
+  - [InclusionListSummaryV1](#inclusionlistsummaryv1)
+  - [InclusionListStatusV1](#inclusionliststatusv1)
+- [Methods](#methods)
+  - [engine_newInclusionListV1](#engine_newinclusionlistv1)
+    - [Request](#request)
+    - [Response](#response)
+    - [Specification](#specification)
+  - [engine_getInclusionListV1](#engine_getinclusionlistv1)
+    - [Request](#request-1)
+    - [Response](#response-1)
+    - [Specification](#specification-1)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Constants
+
+| Name | Value |
+| - | - |
+| `INCLUSION_LIST_MAX_GAS` |  `uint64(4194304) = 2**22` |
```

Comment:
super pedantic, but this could be a u32 right?

Reply:
No it should be u64, since the gas limit is u64 as well (sure we won't exceed u32 anytime soon, but the spec should be consistent)

Reply:
I see, makes sense